docs/_static/custom.js

```diff
@@ -33,4 +33,24 @@ $(document).ready(function () {
     $("option[value='" + selected + "']").attr("selected", "selected");
     window.location = document.URL.replace(current, selected);
   });
+
+  // Cookbook Onboard (AI Assistant). 
```

Comment:
I don't see anywhere that controls for following text:

<img width="842" alt="Screenshot 2024-08-01 at 16 38 42" src="https://github.com/user-attachments/assets/7684e301-ac4b-4566-9d84-6cc68774e932">

`ApeWorX` does not have the correct casing, and where it is coming from (?) is mysterious...

Reply:
Sorry for taking so long to reply! We have a CMS website where you can customize suggestions, view questions asked and the bot's responses, configure and update data sources, and much more! If you'd like an invite link, just reach out to us at hello@cookbook.dev from your corporate email, or please ask your colleague Chris (whom we chat with on Telegram) to add you to the chat :)